core/src/main/res/values/strings.xml

```diff
@@ -245,7 +245,7 @@
     <string name="download_error_forbidden">Forbidden</string>
     <string name="download_canceled_msg">Download canceled</string>
     <string name="download_canceled_autodownload_enabled_msg">Download canceled\nDisabled <i>Auto Download</i> for this item</string>
-    <string name="download_report_title">Downloads completed with error(s)</string>
+    <string name="download_report_title">Downloads completed</string>
```

Comment:
I think there should be two strings. One for auto downloads completed and one for failed downloads.

Reply:
Changed.